src/middlewared/middlewared/plugins/jail_freebsd.py

```diff
@@ -455,7 +459,10 @@ def defaults(self, options):
             index = plugins_obj.retrieve_plugin_index_data(plugins_obj.git_destination)
 
         if options['plugin'] not in index:
-            raise CallError(f'{options["plugin"]} not found')
+            raise CallError(
+                f'{options["plugin"]} not found, possible reasons are plugin repository\'s '
+                f'INDEX file is missing or {options["plugin"]} plugin\'s manifest file might be missing.'
```

Comment:
Too technical error message for end user. Could we say something along the lines of plugin repository is corrupted? What do you think?